draft-ietf-quic-transport.md

```diff
@@ -1854,6 +1902,49 @@ alive when a stream is open. The default is to send a PING frame after 15
 seconds of quiescence. A PING frame has no additional fields.
 
 
+## NEW_CONNECTION_ID Frame {#frame-new-connection-id}
+
+A server sends a NEW_CONNECTION_ID to provide the client with alternative
+connection IDs that can be used to break linkability when migrating connections
+(see {{migration-linkability}}).
+
+The NEW_CONNECTION_ID is as follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|       Sequence (16)           |
```

Comment:
8 bits ought to be enough - we aren't expecting > 255 migrations

... and this can wrap! I suppose you're trying to support a general scheme that supports up to 1 connection ID per packet. 16 bits would limit the BDP to 32K packets in this case (or 32MB, or a ~2Gbps x 100ms path), which is probably fine.